test/data/validation/destinations/snowpipe_streaming.json

```diff
@@ -177,39 +177,6 @@
     "result": false,
     "err": ["database must match pattern \"(^\\{\\{.*\\|\\|(.*)\\}\\}$)|(^env[.].+)|^(.{1,100})$\""]
   },
-  {
```

Comment:
@shekhar-rudder I see this test data file still has field warehouse. Should these be removed? I am guessing any additional field in test data, not part of config does not impact usage in test code?

Reply:
Having `warehouse` in the config is one of the scenario we should be testing. We don't want existing destinations to fail when they are updated next time with the error "'warehouse' field is not supported"

Reply:
Got it. There would be very few snowpipe destination. Can plan to manually update config.